lib/ts/recipeImplementation.ts

```diff
@@ -308,6 +309,16 @@ export default function RecipeImplementation(recipeImplInput: {
                     return domain === normalisedsessionDomain;
                 }
             }
+        },
+
+        updateClientClockDeviation: function (clientClockDeviationInMillis: number): void {
```

Comment:
What we want the users to customize is how we get the clock skew from the response. Ideally, this should get:
- the access token payload (which will include the `iat`)
- the response object if possible while being consistent across interceptors, or the headers at least.

It should return the detected clock skew as a number, which we will set in the `DateProvider`

Reply:
done